Bruin Bite/Supporting Files/LoginAlertPresentable.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  LoginAlertPresentable.swift
+//  Bruin Bite
+//
+//  Created by Samuel J. Lee on 4/22/19.
+//  Copyright © 2019 Dont Eat Alone. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+// To use this protocol, make a segue with identifier "login"
```

Comment:
Fancy! <3 But not entirely sure we want to do it this way.. more on this later but for now this should be fine!